go-controller/pkg/node/node_ip_handler_linux.go

```diff
@@ -146,6 +147,10 @@ func (c *addressManager) Run(stopChan <-chan struct{}, doneWg *sync.WaitGroup) {
 					c.OnChanged()
 				}
 			case <-addressSyncTimer.C:
+				// skip periodic sync since there is no egress or ingress IP rotation in single node
```

Comment:
wouldnt it make more sense to skip the entire address manager Run funciton? The address manager init calls sync, which I think sets the initial addresses for the host.

Reply:
@trozet I just realized that the dynamic node ip feature relies on the addressManager to detect the IP change. This feature is required for single node deployment. 
This is the node ip change PR: https://github.com/ovn-org/ovn-kubernetes/pull/3131.
we may still need the addressManager to act on node IP changes, but not periodical update (which was added to fix the egress IP bug).